ext/mysqlnd/mysqlnd_wireprotocol.c

```diff
@@ -267,6 +267,24 @@ mysqlnd_read_header(MYSQLND_PFC * pfc, MYSQLND_VIO * vio, MYSQLND_PACKET_HEADER
 		pfc->data->packet_no++;
 		DBG_RETURN(PASS);
 	}
+	// @see https://dev.mysql.com/worklog/task/?id=12999
+	if (header->size > 0) {
+		DBG_ERR_FMT("Logical link: try reap pending data. Packet size=%zu", header->size);
+		zend_uchar *buf = emalloc(header->size);
```

Comment:
mysqlnd has an unusual wrapper for memory operation functions. I believe in this case this should have been `mnd_emalloc`.

Reply:
OK